src/daemon/ca/rc.rs

```diff
@@ -52,6 +52,8 @@ pub struct ResourceClass {
     parent_rc_name: ResourceClassName,
 
     roas: Roas,
+
+    #[serde(skip_serializing_if = "AspaObjects::is_empty", default)]
```

Comment:
This field was not previously omitted from the serialization. Will the deserializing party handle this difference?

Reply:
yes, because of default. I think it's better to remove this if empty to lower the noise. Few users will use ASPA at this point and the presence might confuse people actually.